ca_tools/scripts/line_action_client.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python
+
+import rospy
+import ca_tools.msg
+import actionlib
+
+
+def line_follower_client():
+
+    client = actionlib.SimpleActionClient(
+        'linefollower', ca_tools.msg.linefollowerAction)
+
+    client.wait_for_server()   #wait until the server is up
```

Comment:
Put all your comments in the previous line.

Reply:
Adressed